pkg/foundation/cluster_manager.go

```
@@ -114,24 +123,42 @@ func GetAddons() ([]*unstructured.Unstructured, error) {
 		addonPath = path.Join(val, addonPath)
 	}
 
-	err := filepath.Walk(addonPath, func(path string, info os.FileInfo, err error) error {
+	root, err := os.OpenRoot(addonPath)
+	if err != nil {
+		return nil, err
+	}
+	defer root.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `root.Close` is not checked

(errcheck)